pdc/apps/module/models.py

```diff
@@ -1,80 +1,62 @@
 #
-# Copyright (c) 2015 Red Hat
+# Copyright (c) 2018 Red Hat
 # Licensed under The MIT License (MIT)
 # http://opensource.org/licenses/MIT
 from django.db import models
 from pdc.apps.package.models import RPM
 
 
-class UnreleasedVariant(models.Model):
-    # Not the variant from compose ... which back references compose
-    variant_id          = models.CharField(max_length=100, blank=False)
-    variant_uid         = models.CharField(max_length=200, blank=False, unique=True)
-    variant_name        = models.CharField(max_length=300, blank=False)
-    variant_type        = models.CharField(max_length=100, blank=False)
-    # variant_version/_release are _not_ distribution versions/releases
-    variant_version     = models.CharField(max_length=100, blank=False)
-    variant_release     = models.CharField(max_length=100, blank=False)
-    # Default to '00000000' for now since this field will only be used once
-    # other tooling is updated to supply this value. Eventually, this should
-    # not have a default.
-    variant_context     = models.CharField(max_length=100, blank=False, default='00000000')
-    active              = models.BooleanField(default=False)
-    koji_tag            = models.CharField(max_length=300, blank=False)
-    modulemd            = models.TextField(blank=False)
-    rpms                = models.ManyToManyField(RPM)
+class Module(models.Model):
+    variant_id  = models.CharField(max_length=100, blank=False)
+    uid         = models.CharField(max_length=200, blank=False, unique=True)
+    name        = models.CharField(max_length=300, blank=False)
+    stream      = models.CharField(max_length=100, blank=False)
+    version     = models.CharField(max_length=100, blank=False)
+    context     = models.CharField(max_length=100, blank=False)
+    type        = models.CharField(max_length=100, blank=False, default='module')
```

Comment:
~~Out of curiosity: what other types can there be for a module?~~ Assuming this is for backwards compatibility with the v1 API (as the comment there suggests), it might be good to mention it here that no new instances should have a type different than *module*.

Reply:
Good point. Will add a comment here.